components/motor/tmcstepper/stepper_motor_tmc.go

```diff
@@ -579,8 +579,8 @@ func (m *Motor) Home(ctx context.Context) error {
 	return m.ResetZeroPosition(ctx, 0, nil)
 }
 
-// GoTillStop enables StallGuard detection, then moves in the direction/speed given until resistance (endstop) is detected.
-func (m *Motor) GoTillStop(ctx context.Context, rpm float64, stopFunc func(ctx context.Context) bool) error {
+// goTillStop enables StallGuard detection, then moves in the direction/speed given until resistance (endstop) is detected.
+func (m *Motor) goTillStop(ctx context.Context, rpm float64, stopFunc func(ctx context.Context) bool) error {
```

Comment:
trust the decision to keep this, but curious why only this motor will still have `goTillStop`? also very minor nit, line 604 change `GoTillStop` in the error to `goTillStop`